app/config.json

```diff
@@ -0,0 +1,484 @@
+{
+	"baseUrl": "http://localhost:5000",
+	"debug": true,
+	"mapUnlockThreshold": 3,
+	"useLocalSessions": true,
+ 	"rooms": {
```

Comment:
Idk, would it be useful to have rooms in a separate json? This config file might get pretty big